src/BusinessLogic/PostalCode/PostalCodeTransformer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Packlink\BusinessLogic\PostalCode;
+
+use InvalidArgumentException;
+use Logeecom\Infrastructure\Logger\Logger;
+
+/**
+ * Class PostalCodeTransformer
+ *
+ * @package Packlink\BusinessLogic\PostalCode
+ */
+class PostalCodeTransformer
+{
+    /**
+     * The map of the country codes and their supported formats.
+     *
+     * The formats in this map adhere to the following rules:
+     *  - "*" marks a character (a letter or a number) in the postal code.
+     *  - Everything else (spaces and dashes) represent actual characters that should be in that position.
+     *
+     * @var array
+     */
+    protected static $map = array(
+        'GB' => array(
```

Comment:
Great bretain has another countries iso codes where this transformation should be applied:
ISLE_MAN = "IM" 
JERSEY = "JE"
GUERNSEY = "GG"

(array would be the same that GB)

Reply:
These ISO codes will be added to the postal code transformer.